src/test/java/com/qaprosoft/carina/demo/mfp/MobileTest.java

```diff
@@ -40,4 +40,18 @@ public void bottomNavigationButtonsTest() {
         AbstractPage plansPage = commonPage.clickBottomNavigatorButton(BottomNavigatorButtons.PLANS);
         Assert.assertTrue(plansPage.isPageOpened(), "Diary page isn't opened");
     }
+
+    @Test
+    @MethodOwner(owner = "Hostiuk")
+    @TestRailCases(testCasesId = "2")
+    @TestLabel(name = "feature", value = {"mobile", "regression"})
+    public void goalCaloriesEqualsRemainCaloriesTest() {
+        LoginPageBase loginPage = initPage(getDriver(), LoginPageBase.class);
+        loginPage.login(R.TESTDATA.get("email"), R.TESTDATA.get("password"));
+
+        DiaryPageBase diaryPage = initPage(getDriver(), DiaryPageBase.class);
```

Comment:
here you should add opening Diary page, as it was said earlier - you can't put it in the method in this case.